static/flow/views/data-set-view.js

```diff
@@ -286,6 +286,14 @@ var DataSetView = function(options) {
         var validvalues = [];
         var validtimestamps = [];
         
+        if(timestamps == null || values == null) {
+            // if we couldn't get a valid timestamp or value object,
+            // something went terribly wrong and sequence is invalid
+            // i.e., sequence has name="" and we tried to read the
+            // contents of the dataset folder as if it were a sequence folder
+            return;
```

Comment:
Maybe log an error here? Otherwise, looks good to me!

Reply:
Hopefully we should never hit this case.  We had one crazy dataset that was made while testing changes (a path which can no longer be reached), but still...  Never say never!